website/blog/2023-09-18-release-1.4.md

```diff
@@ -0,0 +1,99 @@
+---
+title: Release Notes - Podman Desktop 1.4
+description: Podman Desktop 1.4 has been released!
+slug: podman-desktop-release-1.4
+authors: [jeffmaury]
+tags: [podman-desktop, release, kubernetes, openshift]
+hide_table_of_contents: false
+image: /img/blog/podman-desktop-release-1.4.webp
+---
+
+import ReactPlayer from 'react-player'
+
+Podman Desktop 1.4 Release! 🎉
+
+This is essentially a bug-fix release to fix various issues in UI, but as usual we also added a few new features.
+
+- **Podman 4.6.2**: Podman 4.6.2 included in Windows and Mac installers
+- **Windows Arm64**: Installers and binaries are now provided for Windows on Arm64
+- **Port range mapping**: a range of port can be now mapped when starting a container
+- **Terminal lifetime**: the terminal for a container is now kept open when user switch to another part of the Podman Desktop UI
+- **Create volume UI**: there is now a `Create volume` button on the `Volumes` pages, allowing easier management of volumes
```

Comment:
After reading below, I'd +1 this again. :) It's easy to fall into saying 'we do X now and we do Y now and we do Z now', but given the context of a release note I think that's clear, and we can remove most 'now's and it would be more readable.